cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/validator/PreCompileValidatorImpl.java

```diff
@@ -203,6 +206,38 @@ public void validateResultsSection(
         }
     }
 
+    @Override
+    public void validateNoDuplicateInputs(List<Input> inputs, Input element) {
```

Comment:
List/<? extends InOutParam/> and InOutParam as parameters for the generic method and you could generate the method based on the second parameter of the generic method